staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -208,7 +208,7 @@ var (
 		flowcontrol.PriorityLevelConfigurationSpec{
 			Type: flowcontrol.PriorityLevelEnablementLimited,
 			Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
-				NominalConcurrencyShares: ptr.To(int32(10)),
+				NominalConcurrencyShares: ptr.To(int32(40)),
```

Comment:
Hmm - with this change we're going up from default total of 245 to 275 without changing the lendable percent.
So technically, for usecases that don't need it, we effectively decrease the capacity by 10%.

Shouldn't we also adjust the LendablePercent here?

Reply:
Interesting suggestion. I have added a commit that increases the lendable percent to 75, thus keeping the unlendable shares at 10 like before this PR. There is a small difference due to the increase in total nominal shares, but I suspect that will be unimportant.

Reply:
Thank you!

> There is a small difference due to the increase in total nominal shares, but I suspect that will be unimportant.

I looked into those numbers - there are changes, for typical values that we use in GKE, I see differences of actually available seats <15%. This isn't negligible, but I also don't know how to assess the risk of such a change.

With borrowing, in overloaded cluster I think that the risk will be mitigated, but I think we need ability to disable this change - so I would like to hide this change behind a feature-gate (though it can be enabled by default).

Reply:
Remember that we have two other config changes being considered now:
1. Segregating events (introduced in #128974);
2. rejiggering to stop working around the lack of borrowing (https://github.com/kubernetes/kubernetes/issues/121982#issuecomment-1933534198)